pkgs/tools/archivers/arc_unpacker/default.nix

```diff
@@ -1,29 +1,23 @@
-{ lib, stdenv, fetchFromGitHub, cmake, makeWrapper, boost, libpng, libjpeg, zlib
-, openssl, libwebp, catch }:
+{ lib, stdenv, fetchFromGitHub, cmake, makeWrapper, boost, libpng, libiconv
+, libjpeg, zlib, openssl, libwebp, catch }:
 
 stdenv.mkDerivation rec {
   pname = "arc_unpacker-unstable";
```

Comment:
```suggestion
  pname = "arc_unpacker";
```

Reply:
I believe the official policy is to put "unstable" in the name per the [documentation](https://github.com/NixOS/nixpkgs/blob/master/doc/contributing/coding-conventions.chapter.md#package-naming-sec-package-naming):

> If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23".

I'm assuming that "name" in this context means `pname`.